codes/1_planning.py

```
@@ -1,8 +1,10 @@
-from openai import OpenAI
+from pydantic_ai import messages as ai_messages, Agent
+import asyncio
 import json
 from tqdm import tqdm
```

Ruff (0.11.9)

`tqdm.tqdm` imported but unused

Remove unused import: `tqdm.tqdm`

(F401)